src/show/components/TextField/index.js

```diff
@@ -44,11 +51,18 @@ const TextField = ({
 }) => {
   const [error, setError] = useState('');
   const [inputType, setInputType] = useState(type);
+  const errorColor = value.trim() === '' ? 'text-red-500' : 'text-blue-500';
+  const [focused, setFocused] = useState(false);
+  const borderColor = focused || value.trim() !== '' ? 'border-blue-400' : error ? 'border-red-500' : 'border-gray';
+
+  const handleBlur = () => {
```

Comment:
move outside of the main component.
For all the function inside any component they should be outside

Reply:
done